@ecomplus/storefront-components/src/js/TheCart.js

```diff
@@ -45,7 +47,9 @@ export default {
     },
     amount: {
       type: Object,
-      default: () => {}
+      default: () => {
+        return {}
+      }
```

Comment:
```suggestion
      default () {
        return {}
      }
```

O ideal é não usarmos arrow function aqui pra manter o `this` como a instância do componente.